.env.example

```diff
@@ -11,7 +11,7 @@ DB_PASSWORD=secret
 
 CACHE_DRIVER=file
 SESSION_DRIVER=file
-QUEUE_DRIVER=sync
+QUEUE_DRIVER=database
```

Comment:
> We could of course use the task scheduler, and just have it run as a cron job in sync mode (by default)?

That's true. We could use the database queue, then schedule the queue work command every minute.